### ISSUE TITLE
Does easy GPU-P work with the Telsa P40

### ISSUE THREAD

**Issue by pcgrubb-it** [jamesstringerparsec/Easy-GPU-PV#365]
Hello All,
first time here so if this is the wrong spot please let me know.
I am using a supermirco server board and the tesla p40.
I can get the card to show under prechecks script. 
but it does say the system is compatible.
this leads me to my question is this gpu-p compatible with tesla p40
I am using the latest drivers and the latest version of windows 10 and tried 11 
so not sure what else to try.
Thanks Paul

**Comment by mitchellurgero** [comment]
What's the issue? Is it not showing up in the VM? Is the script having issues transferring drivers?

**Comment by sapere-aude-incipe** [comment]
Yes it is compatible, I am using a P40 in HyperV. Let me know if you need help debugging your issue and I can try to help, maybe share my process etc.

**Comment by DeepCooler** [comment]
I have an rtx A5000. Do you know how to solve the problem?

<img width="852" height="172" alt="Image" src="https://github.com/user-attachments/assets/a904721c-3e77-4b9d-ba57-b50d59310392" />
